src/utils/beneficial.owner.other.ts

```diff
@@ -129,6 +131,8 @@ const setBeneficialOwnerData = (reqBody: any, id: string): ApplicationDataType =
     : {};
   data[StartDateKey] = mapFieldsToDataObject(reqBody, StartDateKeys, InputDateKeys);
 
+  data[CeasedDateKey] = reqBody["is_still_bo"] === '0' ? mapFieldsToDataObject(reqBody, CeasedDateKeys, InputDateKeys) : {};
```

Comment:
Don't we need to check if the BO key still exists before checking it's value? The register journey doesn't have this key

Reply:
A unit test will prove this assumption.

Reply:
I'll add a unit test.

Reply:
I've reverted the adding back of ceased dates. Which can be done in other open review.